config-templates/shopify/customer.yaml

```diff
@@ -0,0 +1,27 @@
+data_source:
+  name: shopify_customers
+  owners:
+    - support@transformdata.io
+  sql_table: SHOPIFY.CUSTOMER ### Update to your opportunities schema.table
+  mutability:
+    type: immutable
+  identifiers:
+    - name: customer
+      type: primary
+      expr: id
+  dimensions:
+    - name: ds
+      type: time
+      type_params:
+        is_primary: True
+        time_granularity: day
+      expr: created_at::date
+  measures:
```

Comment:
would this table be able to have a metric for `unique_customers` and then we can have cumulative metric examples showing WAU etc?